CONTRIBUTING-students.md

```diff
@@ -21,7 +21,7 @@
 ## Getting Started Early
 
 The 2017 student application window is March 20th to April 3rd.
-[The Python Software Foundation Wiki](https://wiki.python.org/moin/SummerOfCode/2016#Students)
+[The Python Software Foundation Wiki](http://python-gsoc.org/#students)
```

Comment:
http://python-gsoc.org/#gettingstarted actually this link is better. It's a step by step guide how students should contact potential org. Maybe we can also copy it here and adopt it to our website instead of just linking to it.